server.go

```diff
@@ -31,6 +31,7 @@ func main() {
 		pflag.PrintDefaults()
 	}
 
+	hostOptions := pflag.StringSliceP("host-option", "O", []string{}, "List of static hosts allowed to connect to (comma separated, name:host:port formatted)")
```

Comment:
Can you also mention that it will replace the host/port box in the UI, or put that into a separate option like `--host-option-display`?

Reply:
Also, can you mention that this will override the default host and port, or make it select the specfied host/port or throw an error when launching easy-novnc?

Reply:
Could you also add this to the feature list on the README, like `- Optionally display a predefined list of host/port combinations.`

Reply:
And, could you add something to the description mentioning that it acts as an additional whitelist?

Reply:
One last thing: this flag will need to be added to the Heroku `app.json`.